src/components/InlineCodeBlock/WrappedText.tsx

```diff
@@ -0,0 +1,59 @@
+import React, {Fragment} from 'react';
+import {StyleProp, TextStyle, View, ViewStyle} from 'react-native';
+import Text from '@components/Text';
+import useThemeStyles from '@styles/useThemeStyles';
+import CONST from '@src/CONST';
+import type ChildrenProps from '@src/types/utils/ChildrenProps';
+
+type WrappedTextProps = ChildrenProps & {
+    /** Style to be applied to Text */
+    textStyles?: StyleProp<TextStyle>;
+
+    /** Style for each word(Token) in the text, remember that token also includes whitespaces among words */
+    wordStyles?: StyleProp<ViewStyle>;
+};
+
+/**
+ * Breaks the text into matrix
+ * for eg: My Name  is Rajat
+ *  [
+ *    [My,' ',Name,' ',' ',is,' ',Rajat],
+ *  ]
+ */
```

Comment:
@cubuspl42 Does it look better for you?
```suggestion
/**
* Breaks the text into matrix
*
* @example
* const text = "My Name is Rajat";
* const resultMatrix = getTextMatrix(text);
* console.log(resultMatrix);
* // Output:
* // [
* //   ['My', ' ', 'Name', ' ', 'is', ' ', 'Rajat'],
* // ]
*/
```
How it will look
![image](https://github.com/Expensify/App/assets/23176449/0aa4a7d8-6b1f-4fa3-8b68-5026ca6ebe7e)